src/migrations/20231106203141_nullable_content_type.ts

```diff
@@ -14,22 +14,14 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-import axios, { AxiosResponse } from "axios";
+import { Knex } from "knex";
 
-import { TransactionId } from "../../types/types";
-import { ContractReader } from "./contractOracle";
+import { NullableContentTypeMigrator } from "../../src/arch/db/migrator";
```

Comment:
This import statement reach up out of the `src` dir to the project root and then back down into the `src` dir. This presents problems after TypeScript compilation, when the compiled JavaScript code in the `/lib` dir attempts to import TypeScript code from the `/src` dir.

Reply:
Causes #4.